logstash-core/lib/logstash/pipeline.rb

```diff
@@ -101,6 +100,10 @@ def ready?
     @ready.value
   end
 
+  def filter_queue_client
+    @filter_queue_client
```

Comment:
I agree that this should be an `attr_reader` ... this is exactly the purpose of attr_reader.